cmd/run.go

```diff
@@ -18,7 +18,7 @@ func newRunCommand() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Args:  cobra.ExactArgs(1),
-		Use:   "run [flags] <file-name>",
+		Use:   "run <file-name>",
```

Comment:
The position of [flags] in the command output is added by the cobra library itself.